config_defaults_inc.php

```diff
@@ -2555,6 +2555,13 @@
 $g_monitor_delete_others_bug_threshold = DEVELOPER;
 
 /**
+ * Access level needed to export filtered bugs
+ * Look in the constant_inc.php file if you want to set a different value.
```

Comment:
This line is present for 6 parameters in the file. 
Do you want me to drop 6 lines? I like consistency :p